p/pahomqtt/pahomqttubi8.5.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : paho.mqtt.golang
+# Version       : v1.2.0
+# Source repo   : https://github.com/eclipse/paho.mqtt.golang.git
+# Tested on     : UBI 8.5
+# Language      : go
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Ankit Paraskar <Ankit.Paraskar@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+# Note :- Script test case will fail due to unavailablity for dependent package to be installed on VM.
```

Comment:
Can you please check the travis logs , it seems to showing a different failure 
**go: cannot find main module, but found .git/config in /home/tester/paho.mqtt.golang
	to create a module there, run:
	go mod init**

Reply:
Hi Priya,

Checking it as on local we are getting below expected error.
fvt_client_test.go:1060: Error on Client.Connect(): network Error : dial tcp 127.0.0.1:1883: connect: connection refused

Reply:
https://app.travis-ci.com/github/ppc64le/build-scripts/builds/254171910